### PR TITLE
Docs update for aws_vpc_ipv6_cidr_block_association inaccuracies

### LINKED ISSUE #22014: Incorrect documentation for aws_vpc_ipv6_cidr_block_association
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version

AWS Provider v3.68.0

### Affected Resource(s)

aws_vpc_ipv6_cidr_block_association

### Terraform Configuration Files

N/A

### Debug Output

N/A

### Panic Output

N/A

### Actual Behavior

The resource documentation states:

> When a VPC is created, a primary IPv6 CIDR block for the VPC must be specified.

This is not correct. There is no such thing as a "primary" IPv6 CIDR block for a VPC and IPv6 CIDR blocks are entirely optional.

The uses of the words "additional" and "further" in the existing description are similarly inappropriate.

### Steps to Reproduce

Read the resource documentation.

### Important Factoids

N/A

### References

https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_VpcIpv6CidrBlockAssociation.html
https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Vpc.html

### PR DESCRIPTION
**Pull request by drewmullen**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #22014

Output from acceptance testing:

```
$ make testacc TESTS=TestAccXXX PKG=ec2

...
```

### CODE REVIEW

File: website/docs/r/vpc_ipv6_cidr_block_association.html.markdown [hunk 1]
```diff
@@ -10,8 +10,8 @@ description: |-
 
 Provides a resource to associate additional IPv6 CIDR blocks with a VPC.
 
-When a VPC is created, a primary IPv6 CIDR block for the VPC must be specified.
-The `aws_vpc_ipv6_cidr_block_association` resource allows further IPv6 CIDR blocks to be added to the VPC.
+When a VPC is created, a primary IPv4 CIDR block for the VPC must be specified.
+The `aws_vpc_ipv6_cidr_block_association` resource an IPv6 CIDR block to be added to the VPC.
```
Comment (johngmyers) [inline website/docs/r/vpc_ipv6_cidr_block_association.html.markdown:14]:
```suggestion
The `aws_vpc_ipv6_cidr_block_association` resource allows IPv6 CIDR blocks to be added to the VPC.
```

File: website/docs/r/vpc_ipv6_cidr_block_association.html.markdown [hunk 2]
```diff
@@ -10,8 +10,8 @@ description: |-
 
 Provides a resource to associate additional IPv6 CIDR blocks with a VPC.
 
-When a VPC is created, a primary IPv6 CIDR block for the VPC must be specified.
-The `aws_vpc_ipv6_cidr_block_association` resource allows further IPv6 CIDR blocks to be added to the VPC.
+When a VPC is created, a primary IPv4 CIDR block for the VPC must be specified.
```
Comment (johngmyers) [inline website/docs/r/vpc_ipv6_cidr_block_association.html.markdown:13]: I don't think this sentence is necessary. It doesn't add to the understanding of the resource being documented.

**Review by johngmyers — CHANGES_REQUESTED** [review]

File: website/docs/r/vpc_ipv6_cidr_block_association.html.markdown (hunk 1 above)
Comment (johngmyers) [reply website/docs/r/vpc_ipv6_cidr_block_association.html.markdown:14]: The IPv4 resource uses the plural form despite each association only adding a single block at a time. But you can use multiple of them to add multiple blocks, so the plural form might be better.

**Review by ewbankkit — APPROVED** [review]
LGTM :rocket:.

**Comment by ewbankkit** [comment]
@drewmullen Thanks for the contribution :tada: :clap:.

**Pull request merged**